train-sensor/src/main/java/hu/bme/mit/train/sensor/TrainSensorImpl.java

```diff
@@ -8,7 +8,7 @@
 
 	private TrainController controller;
 	private TrainUser user;
-	private int speedLimit = 5;
+	private int speedLimit = 6;
```

Comment:
what if next it changes to 6.5 next time? maybe it should be a float?

Reply:
no thats stupid